### ISSUE TITLE
property range violation for 'formation of anatomical boundary' GO:0048859

### ISSUE THREAD

**Issue by balhoff** [geneontology/go-ontology#22537]
[formation of anatomical boundary](http://purl.obolibrary.org/obo/GO_0048859) has logical definition:

```
'anatomical structure formation involved in morphogenesis'
 and ('results in formation of' some 'non-material anatomical boundary')
```

However the range of [results in formation of](http://purl.obolibrary.org/obo/RO_0002297) is [connected anatomical structure](http://purl.obolibrary.org/obo/CARO_0000003) which is a material structure.

@cmungall do you think this is an RO issue? Or is there a different term we can use here?

**Comment by cmungall** [comment]
OK, a lot wrong with this term

The definition is

_The process in which the limits of an anatomical structure are generated. An anatomical structure is any biological entity that occupies space and is distinguished from its surroundings. Anatomical structures can be macroscopic such as a carpel, or microscopic such as an acrosome._

Is this a useful grouping? I think this term should be in do-not annotate

looking at the mix of terms underneath

![image](https://user-images.githubusercontent.com/50745/145294317-bdc867f4-272c-42c3-9181-e9d9bbd1f8de.png)

many of these are what uberon calls anatomical junctions. E.g

id: UBERON:0003052
name: midbrain-hindbrain boundary
def: "The part of the brain that is the morphological boundary between the midbrain and hindbrain and that is the location of an organizing center which patterns the midbrain and hindbrain primordia of the neural plate." [GO:0021555]
intersection_of: UBERON:0007651 ! anatomical junction
intersection_of: overlaps UBERON:0001891 ! midbrain
intersection_of: overlaps UBERON:0002028 ! hindbrain

So I think changing to junction is probably best, then we can close

but the term troubles me, genes maing gradients boundaries etc are likely important in developmental biology but it seems there should be more genes involved

![image](https://user-images.githubusercontent.com/50745/145295970-c63baf04-174d-442a-a574-5f82fb3968f2.png)

experimental

![image](https://user-images.githubusercontent.com/50745/145297261-bf5297cc-fb7b-4d47-a9d9-401ea9eb659f.png)


this feels like a case of terms being added one-off to support a single annotation without consistent population

**Comment by pgaudet** [comment]
Hi @cmungall 

I agree this is a strange grouping class. Also, the definition is quite similar to 'pattern specification process': 'Any developmental process that results in the creation of defined areas or spaces within an organism to which cells respond and eventually are instructed to differentiate.' 

Merge?

**Comment by pgaudet** [comment]
Hi @hattrill 

These terms have no annotations: 


*  **GO:0007358   establishment of central gap gene boundaries** - def: Specification of the borders of central gap gene expression mediated largely by the effects of other gap genes; in insects this is exemplified by knirps repression of Kruppel.
*  **GO:0007361    establishment of posterior gap gene boundaries:** Specification of the borders of posterior gap gene expression mediated largely by the effects of other gap genes; in insects this is exemplified by hunchback and tailless repression of knirps.
*  **GO:0007364   establishment of terminal gap gene boundary:** Specification of the borders of terminal gap gene expression mediated largely by the effects of other gap genes. 

They seem to be intended to group specific transcription factors, without specifying  the developmental processes in which they are involved; are these needed? 

Thanks, Pascale

**Comment by hattrill** [comment]
Not needed - this just describes TF feedback in boundary specification, which can be described in other ways - please obsolete.

**Comment by pgaudet** [comment]
Thanks @hattrill - I opened a separate ticket for these obsoletions: #22546

**Comment by balhoff** [comment]
This will be fixed by #22593.
